packages/perseus/src/multi-items/__tests__/multi-renderer_test.js

```diff
@@ -136,7 +136,7 @@ describe("multi-item renderer", () => {
             // selected, a value is entered). You can see the result of this in the `choiceStates`
             // array in the captured state below where the choice at index 2 has
             // `"selected": true` (instead of false) and the input-number has a `currentValue`.
-            userEvent.click(screen.getAllByRole("radio")[2]); // Correct
```

Comment:
In retrospect, this is a good change. The `radio` role is supposed to always be a mutually exclusive selection (our multi-select answer mode would violate this). 

> The radio role is one of a group of checkable buttons, in a radiogroup, where **_no more than one_** of the buttons can be checked at a time.
— https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radio_role

Reply:
If multiple items can be selected, it feels like we should be using the `checkbox` role as opposed to the `button` role.